src/header.js

```diff
@@ -0,0 +1,14 @@
+export default function Header({ $target }) {
+  const $div = document.createElement("div");
+  $div.setAttribute("class", "header");
+  const $header = document.createElement("h2");
+
+  $target.appendChild($div);
+  $div.appendChild($header);
+
+  this.render = () => {
+    $header.innerHTML = `
+    Notion`;
+  };
```

Comment:
```suggestion
   $header.innerHTML = `Notion`;
  };
```
엄청 사소한 부분이지만 한줄로 하는 것이 어떨까용 ㅎ